backend/SyncReverseProxy/ProxyKernel.cs

```diff
@@ -40,6 +41,15 @@ public static class ProxyKernel
             .LoadFromConfig(reverseProxyConfig)
             .AddTransforms(context =>
             {
+                context.AddRequestTransform(async transformContext =>
```

Comment:
nice, I think I'll probably refactor this a bit, at the very least the resumable api is structured different so it won't be in a route value but this is a good starting point.